versioned_docs/version-2.0-2.4/pages-for-subheaders/rke-add-on.md

```diff
@@ -2,5 +2,9 @@
 title: RKE Add-On Install
 ---
 
+<head>
+  <link rel="canonical" href="https://ranchermanager.docs.rancher.com/pages-for-subheaders/about-rke1-templates"/>
+</head>
+
```

Comment:
```suggestion
```

I don't think there's an equivalent concept to link to in the latest docs. The first thing on the pages linked to on line 9 and 10 is `Important: RKE add-on install is only supported up to Rancher v2.0.8`. For now, let's leave such pages to self-reference until we find a better solution. In this case, this page is about using RKE to install Rancher whereas the proposed link is about downstream RKE clusters so using that as a canonical link is misleading.